src/material/schematics/ng-generate/mdc-migration/rules/components/util.ts

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
It'd be good if we could rename this file to something a bit more descriptive - [go/no-util](http://go/noutil)

Reply:
Agreed 👍 seems like a `test-setup-helper` or something (which is still somewhat close to util though 🤷 )

Reply:
This file should also be part of the test `ts_library` only I think, part of `testonly = True`